src/main/java/org/gridsuite/study/server/repository/LoadFlowResultEntity.java

```diff
@@ -47,7 +47,7 @@ public LoadFlowResultEntity(boolean ok, Map<String, String> metrics, String logs
 
     // we never need to access these without loading the study, and the number of items is small (roughly 10), so we can use ElementCollection
     @Column(name = "componentResults")
-    @CollectionTable(foreignKey = @ForeignKey(name = "loadFlowResultEntity_componentResults_fk"))
+    @CollectionTable(foreignKey = @ForeignKey(name = "loadFlowResultEntity_componentResults_fk"), indexes = @Index(columnList = "loadflowresultentity_id"))
```

Comment:
Set a name of the index ?
`loadFlowResultEntity_componentResults_loadFlowResultId_index` ?